.github/actions/install-linux-dependencies/action.yaml

```diff
@@ -27,7 +27,7 @@ runs:
           if: runner.os == 'Linux'
           run: |
               sudo apt-get update
-              sudo apt-get install libxcb-shape0-dev libxcb-xfixes0-dev libxkbcommon-dev libudev-dev libinput-dev libfontconfig-dev ${{ inputs.extra-packages }}
+              sudo apt-get install libxcb-shape0-dev libxcb-xfixes0-dev libxkbcommon-dev libudev-dev libinput-dev libfontconfig-dev libgtk-3-dev ${{ inputs.extra-packages }}
```

Comment:
Because we do the test with --all-features, thins brings muda in even if we do not use it on linux.

Alternative is to put the muda dependency behind a `target.'cfg`

Reply:
> Alternative is to put the muda dependency behind a `target.'cfg`

That sounds like the way to go then. I mean, didn't we conclude that we can't test this on Linux anyway because of gtk? Or is there any other downside?